jobs/payment-jobs/tests/jobs/test_ejv_partner_partial_refund_distribution_task.py

```diff
@@ -0,0 +1,90 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to assure the CGI EJV Job.
+
+Test-Suite to ensure that the CgiEjvJob is working as expected.
+"""
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import pytest
+from flask import current_app
+from freezegun import freeze_time
+from pay_api.models import CorpType as CorpTypeModel
+from pay_api.models import DistributionCode, EjvFile, EjvHeader, EjvLink, FeeSchedule, db
+from pay_api.utils.enums import DisbursementStatus
+
+from tasks.ejv_partner_distribution_task import EjvPartnerDistributionTask
+
+from .factory import (
+    factory_create_direct_pay_account, factory_distribution, factory_distribution_link, factory_invoice,
+    factory_invoice_reference, factory_payment, factory_payment_line_item, factory_receipt)
+
+
+@pytest.mark.parametrize('client_code, batch_type', [('112', 'GA'), ('113', 'GI'), ('ABC', 'GI')])
+def test_partial_refund_disbursement_for_partners(session, monkeypatch, client_code, batch_type):
```

Comment:
Where are the partial refunds being created?

Reply:
I updated the unit test